python/llm/example/transformers/transformers_int4/mpt/README.md

```diff
@@ -0,0 +1,39 @@
+# MPT
```

Comment:
1. Install
2. Config
3. Run
2.1 Client
2.2 Server

Reply:
Have adjusted the structure as required :)